executor_manager/tasks/task_processor.py

```
@@ -110,8 +110,14 @@ def _process_single_task(self, task):
             tuple: (result dict, success bool)
         """
         task_id = task.get("task_id", -1)
+        subtask_id = task.get("subtask_id", -1)
```

Ruff (0.14.13)

Local variable `subtask_id` is assigned to but never used

Remove assignment to unused variable `subtask_id`

(F841)